alpha-blog/db/migrate/20210513121425_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :username
```

Comment:
Please add validation here as well. For presence & uniqueness.
This validation is database validation.
Please read up, think then tell me why we need validation here as well.